data-structures-stacks/max-value.js

```diff
@@ -0,0 +1,13 @@
+export default function maxValue(stack) {
+  let count = stack.peek();
```

Comment:
While this technically works, `count` probably isn't the best name for the maximum value.